odoo/addons/base/views/res_partner_views.xml

```diff
@@ -208,10 +208,16 @@
                         <group>
                             <field name="function" placeholder="e.g. Sales Director"
                                 attrs="{'invisible': [('is_company','=', True)]}"/>
-                            <field name="phone" widget="phone"/>
-                            <field name="mobile" widget="phone"/>
+                            <div id="base_phone">
```

Comment:
Please don't add unnecessary div. Form view should stay as standard as possible: label, field, as few div as possible. Moreover you may break display as a field has different rendering depending on group, sheet, ... which easily breaks when adding div layers.

Reply:
Thanks for the detailed explanation, I will look into overriding the view instead then :slightly_smiling_face: